### PR TITLE
Project initialisation: Configuration of gatsby

### PR DESCRIPTION
**Pull request by mamadOuologuem**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "import/core-modules": ["react", "config"]
+  },
+
+  "rules": {
+    "func-names": 0,
+    "indent": [2, 2, {"SwitchCase": 1}],
+    "max-len": ["error", 120],
+    "no-confusing-arrow": 0,
+    "no-param-reassign": 0,
+    "no-var": 0,
```
Comment (bencpeters) [inline .eslintrc]: Let's enable this option - prefer using `let`/`const` at this point

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ovio-site-gatsby",
+  "description": "Ovio website",
+  "version": "0.1.0",
+  "private": true,
+  "author": "Mamadou Ouologuem <mamadou.ouologuem02@gmail.com>",
+  "license": "GNU GENERAL PUBLIC LICENSE",
```
Comment (bencpeters) [inline package.json]: Let's not add this until/if we open-source it - for now we're not necessarily using this license.

File: src/components/header.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import PropTypes from 'prop-types';
+
+const Header = ({ siteTitle }) => (
+  <header
+    style={{
+      background: 'rebeccapurple',
```
Comment (bencpeters) [inline src/components/header.js]: I'd prefer not to do styles inline to the extent that we can. If we're going to use Material UI, let's use something like Material UI's [styles API](https://material-ui.com/styles/basics/) to do this.

File: src/components/layout.css [hunk 4]
```diff
@@ -0,0 +1,622 @@
+html {
```
Comment (bencpeters) [inline src/components/layout.css]: Where is this CSS file from? It has a lot of styles in it that we're not using (yet), is it from a framework? I'd prefer to use something well-established that we can theme...

File: package.json (hunk 2 above)
Comment (mamadOuologuem) [reply package.json]: Okay i will remove it 

I put it because Eric in is initial push, pushed the `Licence`

File: src/components/header.js (hunk 3 above)
Comment (mamadOuologuem) [reply src/components/header.js]: Go it :)

I installed `styled-components` ([link](https://www.styled-components.com/)). It is like Material UI ... 
If you prefer Material UI I can change

File: src/components/layout.css (hunk 4 above)
Comment (mamadOuologuem) [reply src/components/layout.css]: Yes it is from the `gatsby` (the default config)

I will clean it on the next PR

**Comment by mamadOuologuem** [comment]
> Is this just a default Gatsby starter from somewhere, or did you set up the `Layout`, etc. components? Are you planning on changing the overall site structure/layout in future commits?

Yes, there is some default config remaining ... I will clean them on the next PR

File: src/components/layout.css (hunk 4 above)
Comment (bencpeters) [reply src/components/layout.css]: No worries, I'm fine starting out with a totally default configuration - just wanted to know what we're getting here ;-)

**Pull request merged**